### PR TITLE
8063 Tworavens Example Manifest Update, Disclaimer

### LINKED ISSUE #8063: Cannot add TwoRavens as an external tool (Dataverse v5.5)
The following error : {"status":"ERROR","message":"contentType is required."} occurs when I run the curl command 
curl -X POST -H 'Content-type: application/json' --upload-file twoRavens.json http://localhost:8080/api/admin/externalTools
with the twoRavens.json downloaded from https://guides.dataverse.org/en/latest/installation/r-rapache-tworavens.html#e-enable-tworavens-in-a-dataverse-installation.
A solution is provided by Jim from the dataverse community (https://groups.google.com/g/dataverse-community/c/yA-g3wlFH48)

jim_solution

### PR DESCRIPTION
**Pull request by djbrooke**
**What this PR does / why we need it**:

In #8063 it was suggested we update the TwoRavens manifest so that it still works with our recent changes to external tools. @pdurbin noted that we should rely on the external developers to maintain these manifests, so along with the update I added a disclaimer and pointed people to the TwoRavens team for support. If this breaks again, we can consider taking more extensive action. 

**Which issue(s) this PR closes**:

Closes #8063

**Special notes for your reviewer**:

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

File: doc/sphinx-guides/source/_static/installation/files/root/external-tools/twoRavens.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "displayName": "TwoRavens",
   "description": "A system of interlocking statistical tools for data exploration, analysis, and meta-analysis.",
+  "toolName": "TwoRavens",
   "scope": "file",
-  "type": "explore",
+  "types": "explore",
```
Comment (pdurbin) [inline doc/sphinx-guides/source/_static/installation/files/root/external-tools/twoRavens.json:6]: Does this work? I think `types` takes an array.

`type` still works for backward compatibility if we want to revert this.

**Review by pdurbin — COMMENTED** [review]
I didn't test it but I don't think the change from `type` to `types` will work as written.

Otherwise, it seems fine but someday I still think we should stop hosting this manifest file for TwoRavens. I know they're grandfathered in.

File: doc/sphinx-guides/source/_static/installation/files/root/external-tools/twoRavens.json (hunk 1 above)
Comment (djbrooke) [reply doc/sphinx-guides/source/_static/installation/files/root/external-tools/twoRavens.json]: good catch, thx. Changed to type in https://github.com/IQSS/dataverse/commit/582f85e95702b04cbebed52a9b97fc29121d824c

**Review by pdurbin — APPROVED** [review]
I didn't test this, but it should work. I'm glad for the note to contact the TwoRavens team as necessary.

**Pull request merged**